samples/generate_html5_console.py

```diff
@@ -103,13 +110,27 @@ def main():
                                              vc_cert)
     vc_fingerprint = vc_pem.digest('sha1')
 
+    if vcenter_version <= 5:
+        console_port = '7331'
+        console_url = "http://" + args.host + ":" + console_port \
```

Comment:
Thanks @derchrisuk for the fix. Can we use the str.format() to format the string here?